File: scripts/fixtures/cap9-deprecated/negative-masked.java
// getConfigValue("ignored-in-line-comment");
/* getConfigValue("ignored-in-block"); */
class Cap9NegativeMaskedJava {

    void ok() {
        String doc = "getConfigValue(\"ignored-in-string\")";
        String doc2 = "call.hasOption(\"ignored\")";
        getConfig().getString("ok");
    }
}

File: scripts/fixtures/cap9-deprecated/negative-masked.swift
// getConfigValue("ignored-in-line-comment")
/* block getConfigValue("ignored") */
let migration = "getConfigValue(\"ignored-in-string\")"
let note = 'hasOption("also-ignored")'

func cap9NegativeMaskedSwift() {
    let _ = getConfig().getString("ok")
    let _ = "\(getConfig().getString(\"ok\"))"
}
